packages/shared/lib/sdk/sync.ts

```diff
@@ -676,9 +685,17 @@ export class NangoAction {
             return;
         }
 
-        const content = stringifyAndTruncateLog(args, 99_000);
+        const [message, payload] = args;
 
-        await this.sendLogToPersist(content, { level, timestamp: Date.now() });
+        await this.sendLogToPersist({
+            type: 'log',
+            level: oldLevelToNewLevel[level],
+            source: 'user',
+            message: String(message),
```

Comment:
message is `any`, and a lot of customers are putting records object or array directly as first argument that's why I added the stringify step; truncating was mostly there to avoid loosing a message if it was too big since customers would have no way to know a message has been skipped

Reply:
I should have documented maybe

Reply:
ok. I will bring back the stringification of message

Reply:
stringifying message and also correctly handle when payload is an array: https://github.com/NangoHQ/nango/pull/2660/commits/0804014faec2914c605ca6b3acf0698c380fb087